core/src/main/java/com/scalar/db/api/ScanBuilder.java

```diff
@@ -349,6 +351,12 @@ public BuildableScanAllWithOngoingWhere where(ConditionalExpression condition) {
       return new BuildableScanAllWithOngoingWhere(this, condition);
     }
 
+    @Override
+    public BuildableScanAllWithWhere where(Set<Set<ConditionalExpression>> conditions) {
```

Comment:
This interface is mainly for internal purposes, but users can see the interface and might use it. So, I didn't use `Conjunction` here.